test/angularCronJobsSpec.js

```diff
@@ -35,7 +79,7 @@ describe('AngularCronJobs', function() {
     it("cron should be set for every hour at 10 past", function() {
         var scope = $rootScope.$new();
         var view = createView(scope);
-        scope.myFrequency = {base: 2, minuteValue: [10]};
+        scope.myFrequency = {base: 2, minuteValues: [10]};
         scope.cron = cronService.setCron(scope.myFrequency);
         $rootScope.$digest();
         expect(scope.cron).toEqual('10 * * * *');
```

Comment:
test will fail if we add the following assertion:
```javascript
expect(view.html()).toContain('<option ng-repeat="value in minuteValues" ng-selected="myFrequency.minuteValues.indexOf(value) >= 0" value="10" class="ng-scope ng-binding" selected="selected">');
```
Notice the `selected="selected"` attribute that's missing in the output.